packager/README.md

```diff
@@ -0,0 +1,45 @@
+## Prerequisites
+
+Acquire the encrypted private key and TLS certificate (these are not in the
+repo), and place in `certs/privkey.pem.enc` and `certs/cert.pem.enc`
+respectively.
+
+## Deploy and run on Google App Engine
```

Comment:
Curious why you chose App Engine Flex over GKE? I think with the latter, the docker image needs to be uploaded to a registry, whereas with App Engine, that doesn't appear necessary. Any other reasons why one would be picked over the other?

Reply:
The main reason I landed on GAE Flex was because I had something similar that already worked there, and I wanted to make the smallest change possible! Also, I have the impression that anything involving Kubernetes is more complicated than I need, although that might not actually be fair…